### PR TITLE
1, karma ipFamilies 2, dcgm-exporter fix dashboard

### PR DESCRIPTION
**Pull request by yogiseven**

### CODE REVIEW

File: .github/linters/ct.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ chart-repos:
   - elastic=https://helm.elastic.co
   - bitnami=https://charts.bitnami.com/bitnami
   - hashicorp=https://helm.releases.hashicorp.com
-  - minio=https://helm.min.io
+  - minio=https://min.io
```
Comment (dongjiang1989) [inline .github/linters/ct.yaml]:
```suggestion
  - minio=https://charts.min.io
```

File: metrics/templates/grafana/dashboards-1.14/dcgm-exporter-gpu.yaml [hunk 2]
```diff
@@ -26,39 +26,48 @@ data:
...
+              "tags": [],
+              "type": "dashboard"
+            },
             "type": "dashboard"
           }
         ]
       },
       "description": "This dashboard is to display the metrics from DCGM Exporter on a Kubernetes (1.13+) cluster",
-      "editable": false,
+      "editable": true,
```
Comment (dongjiang1989) [inline metrics/templates/grafana/dashboards-1.14/dcgm-exporter-gpu.yaml]:
```suggestion
      "editable": false,
```

File: metrics/templates/grafana/dashboards-1.14/dcgm-exporter-gpu.yaml [hunk 3]
```diff
@@ -26,39 +26,48 @@ data:
...
-              "asDropdown": true,
-              "icon": "external link",
-              "tags": [],
-              "targetBlank": true,
-              "title": "",
-              "type": "dashboards"
-            }
-          ],
+      "id": 43,
+      "links": [],
```
Comment (dongjiang1989) [inline metrics/templates/grafana/dashboards-1.14/dcgm-exporter-gpu.yaml]: Please links label

File: metrics/templates/grafana/dashboards-1.14/dcgm-exporter-gpu.yaml [hunk 4]
```diff
@@ -797,11 +840,9 @@ data:
         ]
       },
       "timezone": "",
-      "title": "Kubernetes / GPU Dashboard(NVIDIA)",
-      "uid": "Oxed_c6Wz",
-      "variables": {
-        "list": []
-      },
-      "version": 1
+      "title": "dcgm-gpu",
```
Comment (dongjiang1989) [inline metrics/templates/grafana/dashboards-1.14/dcgm-exporter-gpu.yaml]:
```suggestion
      "title": "Kubernetes / GPU Dashboard(NVIDIA)",
```

**Comment by dongjiang1989** [comment]
Just some nits @yogiseven

**Comment by dongjiang1989** [comment]
@imlense  PTAL

File: metrics/templates/grafana/dashboards-1.14/dcgm-exporter-gpu.yaml (hunk 3 above)
Comment (dongjiang1989) [reply metrics/templates/grafana/dashboards-1.14/dcgm-exporter-gpu.yaml]: > Please add links label

Comment (yogiseven) [reply metrics/templates/grafana/dashboards-1.14/dcgm-exporter-gpu.yaml]: resloved

File: metrics/charts/karma/templates/service.yaml [hunk 5]
```diff
@@ -14,6 +14,11 @@ metadata:
   annotations:
 {{ toYaml .Values.service.annotations | indent 4 }}
 spec:
+{{- if ge (.Capabilities.KubeVersion.Minor|int) 20 }}
+  ipFamilies:
+{{ toYaml .Values.service.ipFamilies | indent 2 }}
```
Comment (dongjiang1989) [inline metrics/charts/karma/templates/service.yaml]:
```suggestion
{{ toYaml .Values.service.ipFamilies | indent 4 }}
```

File: metrics/templates/grafana/dashboards-1.14/dcgm-exporter-gpu.yaml (hunk 2 above)
Comment (imlense) [reply metrics/templates/grafana/dashboards-1.14/dcgm-exporter-gpu.yaml]: editable must be setting false

File: metrics/templates/grafana/dashboards-1.14/dcgm-exporter-gpu.yaml (hunk 4 above)
Comment (imlense) [reply metrics/templates/grafana/dashboards-1.14/dcgm-exporter-gpu.yaml]: title info change it

**Review by imlense — COMMENTED** [review]
change nits

File: metrics/templates/grafana/dashboards-1.14/dcgm-exporter-gpu.yaml (hunk 2 above)
Comment (dongjiang1989) [reply metrics/templates/grafana/dashboards-1.14/dcgm-exporter-gpu.yaml]: Please Re-check it @yogiseven

Comment (yogiseven) [reply metrics/templates/grafana/dashboards-1.14/dcgm-exporter-gpu.yaml]: resloved

**Comment by imlense** [comment]
LGTM

**Pull request merged**